framework/src/media/codec/audio_encoder.h

```diff
@@ -80,7 +82,7 @@ struct audio_encoder_s {
  * @param  enc_ext : Pointer to encoder external struct, ex: opus use opus_enc_external_t
  * @return 0 on success, otherwise, return -1.
  */
-int audio_encoder_init(audio_encoder_p encoder, size_t rbuf_size, audio_type_t audio_type, void *enc_ext);
+int audio_encoder_init(audio_encoder_p encoder, size_t rbuf_size, int audio_type, void *enc_ext);
```

Comment:
Can you use typedef to define audio_type_t?

Reply:
`audio_type_t` is already defined in MediaType.h